### ISSUE TITLE
Inline Help trigger is not focusable

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1008]
**Describe the bug**
It is not possible to see the inline help message with keyboard-only navigation. This is an accessibility issue

> What version of the Fundamental Library Styles are you using?
v0.9.0

**To Reproduce**
1. Go to https://sap.github.io/fundamental-styles/components/inline-help.html
2. Try to navigate to the trigger button by tabbing through the page.
3. This is not possible

**Expected behavior**
Users should be able to trigger the quick-help dialog using keyboard-navigation. Use HTML markup that supports semantic tab-index.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81749346-53af7380-9460-11ea-9aca-1b22078185a0.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by prsdthkr** [comment]
Fixed.
